lib/dry/system/rails/version.rb

```diff
@@ -3,7 +3,7 @@
 module Dry
   module System
     module Rails
-      VERSION = '0.2.0'
+      VERSION = '0.2.1.pre'
```

Comment:
Why not `0.3.0`?

Reply:
Good question. I don’t know what the release process looks like for the other gems in the eco system. 

Is that documented anywhere?

Reply:
We just follow semver. Gems below 1.0 don’t need pre-release suffixes as they are beta/unstable.

It should be documented. I was actually supposed to do that 😅

Reply:
OK no worries.